### DISCUSSION TITLE
Counting lines in files without a file extension?

### DISCUSSION THREAD

**Discussion by bes** [XAMPPRocky/tokei#1110] [Q&A]
Hi! I have a bunch of files, used for tests, that don't have any file extension. Is it possible to count those files with tokei or do I have to add file extensions?

**Comment by XAMPPRocky** [comment]
Thank you for your question! It depends, tokei can read files without extensions, if they have a well known name that tokei can use to pick up on its syntax (e.g. `Dockerfile`).
